.circleci/config.yml

```diff
@@ -1097,7 +1100,13 @@ jobs:
               -DBUILD_COMPONENTS_TESTING=ON \
               ~/datadog/components
             make -j all
-            make test
+            ctest --output-on-failure
+
+            # channel is non-deterministic, so run tests a few more times. At
+            # the moment, Catch2 tests are not automatically adding labels, so
+            # run all tests:
+            # https://github.com/catchorg/Catch2/issues/1590
+            ctest --output-on-failure --repeat until-fail:9
```

Comment:
2 questions here.

1. is line 1103 a leftover?
2. `make test` used to add the `ctest`'s [flag](https://cmake.org/cmake/help/latest/manual/ctest.1.html#:~:text=--force--,new,--ctest-process) `--force-new-ctest-process`. Should we honor that?

Looking at the generated `Makefile`

```
test:
        @$(CMAKE_COMMAND) -E cmake_echo_color --switch=$(COLOR) --cyan "Running tests..."
        /opt/cmake/3.20.1/bin/ctest --force-new-ctest-process $(ARGS)
```
it seems to me that you could obtain the same by `make test ARGS="--output-on-failure --repeat until-fail:9"`

Reply:
Updated as per your feedback.